lighthouse-core/report/html/third_party/details-element-polyfill.js

```diff
@@ -0,0 +1,295 @@
+/*
```

Comment:
I know there's a thing with copying the report renderer files around, but is it possible to support this in the top-level `third-party/`? It should really be there (it makes it very clear its relationship to lighthouse)

Reply:
also need a license file in parallel to it, and ideally a readme that says the source commit and lists the modifications made

Reply:
I'll do these as a followup. It adds too much complexity to the roll right now.